RELEASE-NOTES.txt

```diff
@@ -1,5 +1,7 @@
 17.3
 -----
+* [**] Fix issue where deleting a post and selecting undo would sometimes convert the content to the classic editor. [#16342]
```

Comment:
This also looks like it doesn't belong

Reply:
Same issue as mentioned here https://github.com/wordpress-mobile/WordPress-iOS/pull/16358#discussion_r618729975